test/__init__.py

```diff
@@ -73,7 +73,7 @@
 # 3. To test our timeout logic by using two different values, eg. by using different
 #    values at the pool level and at the request level.
 SHORT_TIMEOUT = 0.001
-LONG_TIMEOUT = 0.01
+LONG_TIMEOUT = 0.1
```

Comment:
Latency using Hypercorn+Quart is more variable than I would like, and I've seen a few read timeout errors before bumping the timeout here.